ddp-workspace/projects/toolkit/package.json

```diff
@@ -17,12 +17,15 @@
     "@nguniversal/express-engine": "8.1.1",
     "@ngx-translate/core": "12.1.2",
     "@ngx-translate/http-loader": "4.0.0",
-    "auth0-js": "9.13.2",
-    "auth0-lock": "11.26.0",
+    "auth0-js": "9.14.0",
+    "auth0-lock": "11.26.3",
+    "bootstrap": "3.4.1",
     "classlist.js": "1.1.20150312",
     "core-js": "2.5.4",
     "ddp-sdk": "1.0.0",
+    "font-awesome": "4.7.0",
     "hammerjs": "2.0.8",
+    "jquery": "3.5.1",
     "kaop-ts": "3.1.2",
```

Comment:
Same question. We need bootstrap, jquey and font-awesome now?

Reply:
All these dependencies are needed for AT.

Reply:
👍